numpy/core/numeric.py

```diff
@@ -401,11 +402,17 @@ def asarray(a, dtype=None, order=None):
         Input data, in any form that can be converted to an array.  This
         includes lists, lists of tuples, tuples, tuples of tuples, tuples
         of lists and ndarrays.
-    dtype : data-type, optional
+    dtype : {None, data-type}, optional
```

Comment:
If None is the default, it never has to be added to the type specifier like this.